### DISCUSSION TITLE
React Suite Tooltip Whisper methods

### DISCUSSION THREAD

**Discussion by AdityaGupta20871** [rsuite/rsuite#3097] [General]
```
import React from 'react';
import "rsuite/dist/rsuite.min.css";
import { useRef } from 'react';
import { Button, Popover, Whisper } from 'rsuite';

export default function App() {
    const whisperRef = useRef(null);
    const handleclick = () => {
        whisperRef.current.open(5000);
      };
	return (
		<div style={{
			display: 'block', width: 700, paddingLeft: 30,marginTop:50
		}}>
			<Whisper ref={whisperRef} 
				trigger="click"
				placement="right"
				speaker={<Popover>Required Text</Popover>}
			>
				<Button appearance="subtle" onClick={handleclick}>Right</Button>
			</Whisper>
			<br />
			<hr />
		</div>
	);
}


```

I am trying to use the open method of the whisper component via using the handleclick function in button component which should delay the opening of Tooltip by 5 milliseconds but the output shows the tooltip component immediately after clicking it .
Kindly check this [documentation](https://rsuitejs.com/components/whisper/#whisper-methods).
Hope to resolve this problem..

**Comment by SevenOutman** [comment]
Hi @AdityaGupta20871 , please [create an issue](https://github.com/rsuite/rsuite/issues/new?template=bug-report.yml) for raising this. You could provide more info to help us locate and resolve the problem.
